src/features/form/containers/RepeatingGroupTableRow.tsx

```diff
@@ -0,0 +1,374 @@
+import React, { useEffect } from 'react';
+
+import { Button, ButtonColor, ButtonVariant, TableRow } from '@altinn/altinn-design-system';
+import { createTheme, makeStyles, TableCell, useMediaQuery } from '@material-ui/core';
```

Comment:
Use `TableCell` from design-system instead of mui.

```suggestion
import { Button, ButtonColor, ButtonVariant, TableCell, TableRow } from '@altinn/altinn-design-system';
import { createTheme, makeStyles, useMediaQuery } from '@material-ui/core';
```